R/estimate_cuminc.R

```diff
@@ -20,34 +20,60 @@
 #' @export
 #'
 #' @examples
-#' estimate_cuminc(
-#'   tidycmprsk::trial,
+#' visR::estimate_cuminc(
+#'   data = tidycmprsk::trial,
 #'   strata = "trt",
 #'   CNSR = "death_cr",
 #'   AVAL = "ttdeath"
-#' ) %>%
```

Comment:
Please keep the full example. The generic functions (e.g. `visr()`, `add_CI()`, etc) don't have cuminc examples, and this is the only place the user will see them in use in the documentation.

Reply:
We will add examples while finishing the test files for the respective functions. Is that OK?

Reply:
There are far fewer users who need competing risks estimation. Let's just keep the one example here showing the results of `estimate_cuminc()` and one visualization.

Reply:
We will keep as discussed during the meeting of 2022 05 02 to keep untill users complain